pgcli/pgclirc

```diff
@@ -170,9 +170,6 @@ arg-toolbar = 'noinherit bold'
 arg-toolbar.text = 'nobold'
 bottom-toolbar.transaction.valid = 'bg:#222222 #00ff5f bold'
 bottom-toolbar.transaction.failed = 'bg:#222222 #ff005f bold'
-literal.string = '#ba2121'
```

Comment:
Apparently it's not obvious that colors set in this section will override colors set by user's choice of `syntax_style`. Could you please add a comment to this section to make it clear?

Reply:
Fixed! :)